test/helm/chart-test.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
 
-# Script to test helm charts for the following:
-## YAML validity
-## chart deployability
-## chart version increment when chart is changed
-## chart installation on Kind cluster
-## run helm tests that make http requests to AEMM service running on the test cluster
+# Script to run E2E tests with Helm and AEMM to test for the following:
```

Comment:
super nit: do you think it's ambiguous with the current wording? Could be interpreted as use helm chart then run e2e tests (the aemm integ tests)? If not, then ignore :D

Reply:
well, if you thought it, then, may be it is :) 
I changed it to `Script to run Helm E2E tests to test for the following:`